form.html

```diff
@@ -0,0 +1,207 @@
+          <ul class="intro-list">
+            <li class="intro-list__item intro-list__item--top">Welcome to the gorgeous</li>
+            <li class="intro-list__item intro-list__item--middle">Sedona</li>
+            <li class="intro-list__item intro-list__item--bottom">"Because the Grand Canyon sucks!"</li>
+          </ul>
+        </div>
+      </header>
+      <main class="main-content">
+        <section class="preamble">
+          <div class="preamble__inner">
+            <h2 class="preamble__header">Leave your feedback</h2>
+            <p class="preamble__paragraph">Help our hotels to become better! Leave a comment about it, also please leave a feedback about the sights you visited</p>
+          </div>
+        </section>
+
+        <form class="form" action="" method="get">
+
+          <section class="form__section">
+            <div class="form__personal">
+              <h2 class="form__section-header">Introduce yourself:</h2>
+              <div class="form__name">
+                <label class="form__label-text" for="name">Name:</label>
+                <input class="form__input-text" id="name" name="name" type="text">
+              </div>
+              <div class="form__surname">
+                <label class="form__label-text" for="surname">Surname:</label>
+                <input class="form__input-text" id="surname" name="surname" type="text">
+              </div>
+              <div class="form__middle-name">
+                <label class="form__label-text" for="second-name">Middle name:</label>
+                <input class="form__input-text" id="second-name" name="second-name" type="text">
+              </div>
+            </div>
+          </section>
+
+          <section class="form__section">
+            <div class="form__contact">
+              <h2 class="form__section-header">Contact Information:</h2>
+              <div class="form__phone">
+                <label class="form__label-tel" for="phone">Contact number:</label>
+                <input class="form__input-tel" name="phone" type="tel">
+                <p class="form__error">Error, double-check the number</p>
+              </div>
+              <div class="form__email">
+                <label class="form__label-email" for="email">Email:</label>
+                <input class="form__input-email" id="email" name="email" type="email">
+              </div>
+            </div>
+          </section>
+
+          <section class="form__section">
+            <div class="form__date">
+              <h2 class="form__section-header">
+                <div class="form__checkin">
+                  <label class="form__label-date" for="checkin">CHECKIN:</label>
+                  <input class="form__input-date" id="checkin" name="checkin" type="date">
+                </div>
+                <div class="form__stay">
+                  <label class="form__label-number-tablet" for="stay">DAYS OF STAY<p class="form__important">*</p></label>
```

Comment:
I think `form__required` is better than `form__important`.
Also add `require` attribute to all the required inputs